snap/snapcraft.yaml

```diff
@@ -18,24 +18,24 @@ source-code: https://github.com//snapcrafters/pyradio
 license: MIT
 version: '0.9.3.7'
 
-architectures:
-  - build-on: amd64
-  - build-on: arm64
-  - build-on: armhf
-
```

Comment:
could you please include a `platforms` block for those supported? ([example](https://github.com/snapcrafters/terraform/blob/6f1c75d137873462cda2ae77d8b5a512b4a1e160/snap/snapcraft.yaml#L34-L39))